core/src/main/java/io/confluent/kafka/schemaregistry/leaderelector/kafka/SchemaRegistryCoordinator.java

```diff
@@ -188,6 +212,18 @@ protected Map<String, ByteBuffer> onLeaderElected(
         leaderKafkaId = kafkaMemberId;
         leaderIdentity = memberIdentity;
       }
+      if (eligible && memberIdentity.isLeader() && !multipleLeadersFound) {
```

Comment:
Can we wrap this block around if (!stickyLeaderElection) as none of this is required in that case.